VideoAPI/Video.API/Controllers/ConsultationController.cs

```diff
@@ -228,19 +230,51 @@ public async Task<IActionResult> StartPrivateConsultationWithEndpointAsync(Endpo
                 _logger.LogWarning(message);
                 return Unauthorized(message);
             }
-            
+
             if (!endpoint.DefenceAdvocate.Trim().Equals(defenceAdvocate.Username.Trim(), StringComparison.CurrentCultureIgnoreCase))
             {
                 var message = "Defence advocate is not allowed to speak to requested endpoint";
                 _logger.LogWarning(message);
                 return Unauthorized(message);
             }
-            
+
             await _videoPlatformService.StartEndpointPrivateConsultationAsync(conference, endpoint, defenceAdvocate);
 
             return Accepted();
         }
-        
+
+        /// <summary>
+        /// End Judge/JoH consultation
+        /// </summary>
+        /// <param name="request">EndConsultationRequest</param>
+        /// <returns>Ok</returns>
+        [HttpPost("end/{roomid}")]
+        [SwaggerOperation(OperationId = "EndJudgeJohConsultationRequest")]
+        [ProducesResponseType((int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> EndJudgeJohConsultationRequestAsync(EndConsultationRequest request)
+        {
+            _logger.LogDebug("EndJudgeJohConsultationRequest");
```

Comment:
Don't need this log anymore since @levibutler has added logging middleware that will do it automatically

Reply:
Removed the logging.